doc/ref_model/chapters/chapter06.rst

```diff
@@ -60,6 +107,58 @@ The Or-Vi and Vi-Vnfm are both specifying interfaces provided by the VIM and the
 Tenant Level APIs
 ~~~~~~~~~~~~~~~~~
 
+<<<<<<< HEAD
```

Comment:
remove

Reply:
Again a half done merging...

Reply:
Fixed in [f39b95f](https://github.com/cntt-n/CNTT/pull/2931/commits/f39b95f9a9cfbd4d02bbb624817269c76e05707e)